libs/api-interfaces/src/lib/aggregate-views/view-models/resources/playlist/playlist.view-model.interface.ts

```diff
@@ -5,6 +5,8 @@ import { IPlaylistEpisode } from './playlist-episode.interface';
 export interface IPlayListViewModel extends IBaseViewModel {
     name: IMultilingualText;
 
+    url?: string;
+
     // TODO establish a view model for episodes
     episodes: IPlaylistEpisode[];
```

Comment:
Each `IPlaylistEpisode` has a `mediaItemUrl` property. You need to flow these through in the detail view to the audio url preview renderer.